### DISCUSSION TITLE
Что? Где? Зачем?

### DISCUSSION THREAD

**Discussion by ezinber** [ezinber/pomo-project#6] [Q&A]
На данный момент имеем: 
- репозиторий **pomo-project**;
- проект в этом же репозитории (он же канбан, отражающий наши телодвижения с кодом. Вкладка **Projects** вверху);
- этот чат (вкладка **Discussions** вверху), который вроде удобный, но если что, можем уйти в какой-нибудь мессенджер;
- проект в [Figma](https://www.figma.com/files/project/29410726/pomo-project?fuid=847967376652770214). Проверьте, все ли могут его редактировать. Кстати, в фигме тоже можно комментить и отвечать, кнопка "пузырь" сверху;

Убедительная просьба не стесняться и выкладывать все идеи, вопросы, претензии и пожелания. Если есть ощущение, что непонятно, с чего вообще начать - это нормально. Мы все тут сами себе тимлиды, поэтому сбиваемся в кучу и пытаемся что-то толковое наклепать

**Comment by nastyaa-l** [comment]
Привет всем. Предлагаю набросать проект в фигме. Могу попробовать начать этим заниматься, если ни у кого нет возражений )

**Reply by ezinber** [reply]
Возражений, естественно, нет. Там есть комментарии, в которых тоже можно обсуждение по каждому элементу вести, если что. Тогда по образцу ориентируемся? Надо скрин закинуть в фигму

**Reply by nastyaa-l** [reply]
Пока искала еще варианты оформления, нашла вот такое приложение https://focustodo.cn/, довольно похожее на нашу идею. Понравился там таймер и наличие разного рода статистики. Немного скринов оттуда, если не хочется устанавливать приложение.
![image](https://user-images.githubusercontent.com/71121357/115766894-2ded5700-a3b1-11eb-994b-38e0d60e0cc3.png)
![image](https://user-images.githubusercontent.com/71121357/115766688-eb2b7f00-a3b0-11eb-923e-8c2809e3fc8f.png)
![image](https://user-images.githubusercontent.com/71121357/115767123-7147c580-a3b1-11eb-9eb7-5266d0ccc189.png)

**Reply by ezinber** [reply]
Мне идея радиального таймера нравится, на скрине он типа "косточками" изображён

**Reply by nastyaa-l** [reply]
Вот, мне тоже это понравилось

**Reply by ezinber** [reply]
где-то я видел такие скрипты, надо распотрошить их будет

**Comment by ezinber** [comment]
Если вдруг кому-то неудобно общение в гите, то давайте выберем мессенджер, в котором всем удобно будет общаться. В слаке, как я понял, нужно новый канал создавать и непонятно, можно ли одновременно в двух сидеть. Мне удобны ВК и Телеграмм

**Reply by nastyaa-l** [reply]
Я за телеграмм)

**Reply by enslit** [reply]
Я тоже за телеграмм

**Reply by enslit** [reply]
Мне кажется, что можно и в слаке сделать канал. Их там можно сколько угодно много делать и общаться так же, в тредах. Телеграм это простыня с перепиской, когда, кто и что писал, будет найти сложно. Просто потрындеть там конечно удобнее в разы, но если нужно будет к чему-то вернуться, то я бы выбрал слак

**Reply by ezinber** [reply]
Попробовал сейчас в слаке сделать канал, он у меня разлогинивается из яндекса, перебрасывает на новую страницу, и там всё по новой: профиль, аватарку ставить. Непонятно, могут ли параллельно два канала нормально работать. В веб версии, во всяком случае

UPD
Хотя вроде параллельно можно, в разных вкладках

**Reply by nyamnyannyak** [reply]
Я тоже за тг

**Reply by ezinber** [reply]
Поскольку нам надо соединить все приложения в одном, канал в слаке всё же [создал](https://join.slack.com/t/pomo-projectworkspace/shared_invite/zt-pcbnjr6e-erEn~DBMxhty0MRXWBxyKA)

Чат в телеграме тоже не помешает, поэтому пусть кто-нибудь организует, мой акк: **@ezinber**

**Reply by nastyaa-l** [reply]
Могу создать чат в тг. Нужны тогда аккаунты еще @enslit  и @nyamnyannyak

**Reply by nyamnyannyak** [reply]
@risa_desu 
тг

**Reply by enslit** [reply]
@prosto_alex
